modules/velero-obtaining-by-downloading.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="velero-obtaining-by-downloading_{context}"]
+= Downloading the Velero CLI tool
+
+You can download and install the Velero CLI tool by following the instructions on the link:https://velero.io/docs/v1.8/basic-install/#install-the-cli[Veleo documentation page].
+
+The page includes instructions for:
+
+* MacOS using Homebrew
+* GitHub
+* Windows using Chocolatey
+
+.Prerequisites
+
+* Access to a Kubernetes cluster, v1.16 or later, with DNS and container networking enabled.
+* `kubectl` installed locally.
+
+.Procedure
+
+. Open a browser and navigate to link:https://velero.io/docs/v1.8/basic-install/#install-the-cli["Install the CLI" on the Verleo website].
+. Follow the appropriate procedure for MacOS, GitHub, or Windows.
+. If possible, download the Velero version appropriate for your version of OADP, according to the table that follows:
```

Comment:
Is "If possible" necessary here? What are the consequences of ignoring this guidance?

Reply:
The Jira says "If you are downloading CLI instead of using alias, the velero CLI tool should ideally be of the same version as the velero server running in the cluster." I'm not aware of anything that might the user from doing so, therefore i've removed "if possible."